.github/workflows/integrationTest.yml

```diff
@@ -431,7 +431,7 @@ jobs:
         uses: nick-invision/retry@v2
         with:
           max_attempts: 3
-          timeout_minutes: 15
+          timeout_minutes: 20
```

Comment:
25 minutes or 30 minutes would be fine by me. What would be the result of bad timeout? As far as I am concerning, none and the test seems not holding on something or leaking something.

Reply:
It is more I don't want to wait for a hanging test to finish. Remember in the otel project we would have tests hanging for 4 hours. I can change this to 30 if you want.

Reply:
Based on what I remembered, its related to create AWS resources which are not on our part; however, that does not seem frequency enough to take a concern. Therefore, 30 minutes would be fine by me and bad case, only waiting for 1 hour and 30 minutes :)